ui/lib/core/addon/templates/components/replication-action-demote.hbs

```diff
@@ -19,7 +19,7 @@
       @text="Demote"
       @color="secondary"
       {{on "click" (action (mut this.isModalActive) true)}}
```

Comment:
For these replication buttons I am not using the exact text because a test iterates through these specific "action" key words.